vaadin-date-time-picker-flow/pom.xml

```diff
@@ -127,12 +127,12 @@
         <dependency>
             <groupId>org.webjars.bowergithub.vaadin</groupId>
             <artifactId>vaadin-time-picker</artifactId>
-            <version>2.0.2</version>
+            <version>2.0.5</version>
         </dependency>
         <dependency>
             <groupId>org.webjars.bowergithub.vaadin</groupId>
             <artifactId>vaadin-combo-box</artifactId>
-            <version>5.0.6</version>
+            <version>5.0.9</version>
```

Comment:
nit: can be updated to `5.0.10`

Reply:
Updated. I'll merge if/when CI is green.